src/backend/joanie/core/factories.py

```diff
@@ -690,7 +690,7 @@ class Meta:
         OrderFactory,
```

Comment:
We should update OrganizationFactory to generate random address I think.

Reply:
I'm not sure, as OrganizationFactory is often nested in others factories, and AddressFactory creates an User.

Reply:
We could something like that 🤔 
`address = factories.SubFactory(Address, user=None)`

Reply:
As we discussed, the address relation has been removed.